src/main/java/edu/greenblitz/robotName/subsystems/Lifter/ILifter.java

```diff
@@ -0,0 +1,18 @@
+package edu.greenblitz.robotName.subsystems.Lifter;
+
+import com.revrobotics.CANSparkMax;
+
+public interface ILifter {
+    public void setPower(double power);
+
+    public void setVoltage(double voltage);
+
+    public void resetEncoder(double position);
+
+    public void stopMotor();
+
+    public void setIdleMode(CANSparkMax.IdleMode idleMode);
+    public void goToPositionByPID(double pos);
```

Comment:
setPosition instead of `goToPositionByPID`

Reply:
we were told to write excextly goToPositionByPID

Reply:
by who?

Reply:
just use goToPosition